cloud-functions/src/utils/webHook.ts

```
@@ -0,0 +1,81 @@
+			return null;
+		}
+	}
+	static async deletePost(postDetails: IPostDelete) {
+		try {
+			return this.sendRequest(EWebhookEvent.POST_DELETED, postDetails);
+		} catch (error) {
+			console.log(error);
+			return null;
+		}
+	}
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)